tests/includes/juju.sh

```diff
@@ -44,9 +49,11 @@ bootstrap() {
         unset BOOTSTRAP_REUSE
     fi
 
+    version=$(juju_version)
+
     START_TIME=$(date +%s)
     if [ -n "${BOOTSTRAP_REUSE}" ]; then
-        echo "====> Reusing bootstrapped juju"
+        echo "====> Reusing bootstrapped juju ($(green "${version}"))"
```

Comment:
Side-note for future improvements: it would be cool if we could also list the provider name when bootstrapping/destroying

Reply:
YES!

Good shout, I'll do that next time.